findAllAnagramsInAString.py

```diff
@@ -0,0 +1,42 @@
+class Solution(object):
+    def findAnagrams(self, s, p):
+        """
+        :type s: str
+        :type p: str
+        :rtype: List[int]
+        """
+        summary = {'a': 0, 'b': 1, 'c': 2, 
```

Comment:
Can write code to auto generate this.
Don't have to type all these.
```
summary = {};
for index in range(ord('a'), ord('z') + 1):
    summary[chr(index)] = index
```
Even you don't remember `ord(a)` is 97.